redaxo/src/core/fragments/core/top.php

```diff
@@ -79,6 +79,13 @@
     <?php echo $this->favicon ? '<link rel="shortcut icon" href="' . $this->favicon . '" />' : '' ?>
 
     <?php echo $this->pageHeader ?>
+    <link rel="stylesheet" href="<?= rex_url::coreAssets('shoelace/dist/themes/light.css') ?>" />
+    <script type="module" src="<?= rex_url::coreAssets('shoelace/dist/shoelace.js') ?>"></script>
```

Comment:
Shoelace einsetzen?

Reply:
wir nutzen bei uns in der firma shoelaec auf dem application level. ich befürchte dass shoelace aktuell noch zu viele BC changes zwischen den ganzen beta versionen hat um es in redaxo einzubetten. da man - wenn man es dann im core updated - immer wieder breaking changes hat für die enduser.

ausgenommen man würde definieren dass der backend admin shoelace direkt nicht nehmen darf sondern nur die redaxo componenten.

sobald shoelace mal ein stable release hatte wären diese bedenken obsolete

Reply:
Direkt sollte man Shoelace nicht nutzen und eigentlich auch nicht Bootstrap. In der idealen Welt stellt Redaxo die Komponenten zur Verfügung, die im Backend angewendet werden. So ließe sich auch am einfachsten das Framework einmal austauschen.

Reply:
Es ist gut, dass es nun shoelace 2.0 stable gibt.

Trotzdem bin ich mir noch unsicher, ob wir shoelace einsetzen sollten.
Es scheint schon sehr sinnvoll aufgebaut zu sein, es ist gut dokumentiert, und macht wohl Spaß, damit zu arbeiten.
Gleichzeitig fehlen aber wohl auch durchaus noch Dinge, für die man Komponenten gebrauchen könnte.

Ich weiß nicht, ob man daher nicht doch eher auf ein vollständigeres Framework setzen sollte, wie eben Bootstrap.

Denke es hat Vorteile, nicht zu viele verschiedene Libs zu benötigen. 

Tue mich bisher schwer, hier eine klare Meinung zu entwickeln. Schwanke immer hin und her.